README.txt

```diff
@@ -1 +1,24 @@
-## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
\ No newline at end of file
+## This repo has the svg artwork for the Backed Community NFT and bunny traits for its avatar.
+
+**How to Earn XP**
+
+**Earn Activity XP**
+
+- As a lender, provide your first loan (+1XP for every 5th)
```

Comment:
What do we think about exponential backoff? I would like to get people a point on their very first of every action. Maybe on your first, fifth, and then every tenth there after?